.eslintrc.json

```diff
@@ -0,0 +1,71 @@
+{
+	// Provide predefined global variables
+	"env": {
+		"browser": true, // browser global variables
+		"es2021": true, // all ECMAScript 2021 globals
+		"node": true // Node.js global variables and Node.js scoping
+	},
+	"extends": "eslint:recommended",
+	"parserOptions": {
+		"ecmaVersion": "latest", // The version of ECMAScript to support — the most recent supported version
+		"sourceType": "module" // The type of JavaScript source code — ECMAScript modules (ESM)
+	},
+	"rules": {
+		// Tabbed indentation (https://eslint.org/docs/latest/rules/indent)
+		"indent": [
+			"error",
+			"tab",
+			{
+				"outerIIFEBody": 0 // Do not indent IIFE body (https://eslint.org/docs/latest/rules/indent#outeriifebody)
+			}
+		],
+		// Unix line endings: \n for LF (https://eslint.org/docs/latest/rules/linebreak-style)
+		"linebreak-style": [
+			"error",
+			"unix"
+		],
+		// Double quotes wherever possible (https://eslint.org/docs/latest/rules/quotes)
+		"quotes": [
+			"error",
+			"double",
+			{
+				"allowTemplateLiterals": true // (https://eslint.org/docs/latest/rules/quotes#allowtemplateliterals)
+			}
+		],
+		// Never omit curly braces around blocks (https://eslint.org/docs/latest/rules/curly)
+		"curly": "error",
+		// Never omit semicolons at the end of statements (https://eslint.org/docs/latest/rules/semi)
+		"semi": [
+			"error",
+			"always"
+		],
+		// Stroustrup brace style for blocks (https://eslint.org/docs/latest/rules/brace-style#stroustrup)
+		"brace-style": [
+			"error",
+			"stroustrup",
+			{
+				"allowSingleLine": true // if (foo) { bar(); } is an example of allowed style of block
+			}
+		],
+		// Never omit whitespace between the function name or function keyword and the opening paren (https://eslint.org/docs/latest/rules/space-before-function-paren)
+		"space-before-function-paren": [
```

Comment:
I tend to use a space before the opening paren when it's a function **definition** and not when it's a function call.
This allows me to be able to find function definitions with simple text search.
Is this about both or only definitions?

Reply:
> I tend to use a space before the opening paren when it's a function definition and not when it's a function call.
This allows me to be able to find function definitions with simple text search.

I really like this convention and follow it all over my code (for a pretty long), thanks to you.

> Is this about both or only definitions?

This is about only definitions. But it doesn't restrict us from adding whitespace before the opening paran when calling a function. So I added a rule that fixes it.

And we have this:

![image](https://user-images.githubusercontent.com/9166277/233024558-2eb8354d-84b3-49c5-8272-63bba62f17d5.png)

Reply:
Love it, thanks!